airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessBuilderFactoryPOC.java

```diff
@@ -97,8 +109,82 @@ public static void testSyncWorkflow() throws IOException, InterruptedException {
     System.exit(0);
   }
 
-  public static void main(String[] args) throws InterruptedException, IOException {
-    testSyncWorkflow();
+  private static void spikeApiAndConfig() throws InterruptedException {
+    Volume configVolume = new VolumeBuilder()
+        .withName("airbyte-config")
+        .withNewEmptyDir()
+        .endEmptyDir()
+        .build();
+
+    VolumeMount pipeVolumeMount = new VolumeMountBuilder()
+        .withName("airbyte-config")
+        .withMountPath("/config")
+        .build();
+
+    var ip = KubePodProcess.getPodIPFuzzyMatch(KubeProcessBuilderFactoryPOC.KUBE_CLIENT, "airbyte-server", "default");
+    LOGGER.info("server pod ip: {}", ip);
+
+    var container = KubePodProcess.writeSourceConfig(ip, UUID.fromString("2e0bb52f-30bd-4f1f-8cc6-fa0382ed66b8"), "config.json", pipeVolumeMount);
+
+    Pod pod = new PodBuilder()
+        .withApiVersion("v1")
+        .withNewMetadata()
+        .withName("test-write-config")
+        .endMetadata()
+        .withNewSpec()
+        .withRestartPolicy("Never")
+        .withContainers(container)
+        .withVolumes(configVolume)
+        .endSpec()
+        .build();
+
+    LOGGER.info("Creating pod...");
+    var podDefinition = KubeProcessBuilderFactoryPOC.KUBE_CLIENT.pods().inNamespace("default").createOrReplace(pod);
+
+    LOGGER.info("Waiting until pod is ready...");
+    KubeProcessBuilderFactoryPOC.KUBE_CLIENT.resource(podDefinition).waitUntilReady(5, TimeUnit.MINUTES);
+  }
+
+  public static void main(String[] args) throws InterruptedException, IOException, ApiException {
```

Comment:
to reproduce this, build the core images then deploy the dev overlays.

after that, build the airbyte-worker image and deploy the sync-worker yaml from resources.